src/extension/content-script/batteries/Mixcloud.ts

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+
+import getOriginData from "../originData";
+import { findLightningAddressInText, setLightningData } from "./helpers";
+
+const urlMatcher = /^https:\/\/www.mixcloud.com\/([^/]+)(\/)?([^/]+)?(\/)?$/;
```

Comment:
are those basically all mixcloud urls?

Reply:
Yes, the Artist page: https://www.mixcloud.com/djcity/
and the Track/Show page: https://www.mixcloud.com/djcity/spinelli/

Reply:
those are three match groups? would checking for the two be enough?

Reply:
Yes, those are the only places where we can find content by creators